src/conversions.jl

```diff
@@ -89,34 +89,18 @@ function astimezone(zdt::ZonedDateTime, tz::FixedTimeZone)
     return ZonedDateTime(zdt.utc_datetime, tz, tz)
 end
 
-function zdt2julian(zdt::ZonedDateTime)
-    datetime2julian(utc(zdt))
-end
+datetime2julian(zdt::ZonedDateTime) = datetime2julian(utc(zdt))
```

Comment:
My preference would be to extend via:
```suggestion
Dates.datetime2julian(zdt::ZonedDateTime) = datetime2julian(utc(zdt))
```
The rest of this package extends this way so I'd like to keep it consistent